hearpreprocess/pipeline.py

```diff
@@ -798,6 +867,19 @@ def run(self):
                 indent=True,
             )
 
+            # Save the slug and the label in as the split metadata
+            audiolabel_df.to_csv(
+                self.workdir.joinpath(f"{split_path.stem}.csv"),
+                index=False,
+            )
```

Comment:
Why are we also saving this as a csv?

Reply:
We need it below to calculate the vocab?

Reply:
yes